l10n/en-US/viewer.ftl

```diff
@@ -414,7 +414,8 @@ pdfjs-editor-comments-sidebar-close-button =
 pdfjs-editor-comments-sidebar-close-button-label = Close the sidebar
 
 # Instructional copy to add a comment by selecting text or an annotations.
-pdfjs-editor-comments-sidebar-no-comments = Add a comment by selecting text or an annotation.
+pdfjs-editor-comments-sidebar-no-comments1 = See something noteworthy? Highlight it and leave a comment.
```

Comment:
The sidebar seems to have a double top/bottom border now, which makes it look like there are two sidebars on top of each other. I assume that's not intended?

<img width="280" height="258" alt="Image" src="https://github.com/user-attachments/assets/997e7110-1337-446b-b187-6ef17a74c43e" />